RecoPixelVertexing/PixelLowPtUtilities/plugins/ClusterShapeHitFilterESProducer.cc

```diff
@@ -71,12 +73,17 @@ ClusterShapeHitFilterESProducer::ReturnType ClusterShapeHitFilterESProducer::pro
     const ClusterShapeHitFilter::Record& iRecord) {
   using namespace edm::es;
 
+  const SiStripLorentzAngle* theSiStripLorentzAngle = nullptr;
+  if (!isPhase2_) {
+    theSiStripLorentzAngle = &iRecord.get(stripToken_);
```

Comment:
Could the `consumes()` call be avoided too?

Reply:
addressed in the last push at 1647c50ac21026b26b264d53973b03a4e4173eb5